src/catala/literate/html.ml

```diff
@@ -153,11 +153,24 @@ let program_item_to_html (i : A.program_item) (custom_pygments : string option)
           | _ -> "#" )
           (pre_html (Pos.unmark a.law_article_name))
     | A.CodeBlock (_, c) | A.MetadataBlock (_, c) ->
+        let date = "\\d\\d/\\d\\d/\\d\\d\\d\\d" in
+        let syms = R.regexp (date ^ "|!=|<=|>=|--|->|\*|\/") in
```

Comment:
We're missing `=` here, in Catala that's the equal comparison and not definition so it should also be rendered in fancy font.

Reply:
Something like ＝compared to the normal = (https://www.fileformat.info/info/unicode/char/ff1d/browsertest.htm)